entrypoint.sh

```diff
@@ -62,6 +71,7 @@ function setup_environment() {
 	declare -a linter_failures
 	declare -a linter_successes
 	declare -a linter_skipped
+	input="/etc/opt/goat/linters.json"
```

Comment:
Seems like a strange place to put this.  Why do we need it here?  I would think we just export a global `INPUT` and rename it `LINTER_CONFIG`

Reply:
Oh, I like that better. I'm used to lumping global vars or static values at the top of an app. Carryover from C#, I'm sure. I'll add this change, though. It flows better from a bash perspective.